source/_components/mikrotik.markdown

```diff
@@ -80,7 +80,7 @@ method:
 arp_ping:
   description: Use ARP ping with DHCP method for device scanning.
   required: false
-  default: false
+  default: true
```

Comment:
I see [here](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/mikrotik/__init__.py#L52) that it's `False`